### PR TITLE
Configurable max-connections + validation-queue-size

### PR DESCRIPTION
**Pull request by bkase**
The hypothesis is that this will significantly reduce the memory
consumption of libp2p_helper

It will be hard to test this "for real" until we try it in production on a big testnet with our community! Let's try it!

### CODE REVIEW

**Comment by nholland94** [comment]
I'm a little worried this will effect our network's ability to scale to larger node counts, but this is worth a shot. 250 seems rather high.

**Review by nholland94 — APPROVED** [review]

**Comment by lk86** [comment]
can we make this configurable at runtime? may be worthwhile to keep our beefy nodes at a high connection count but ship with a lower default for the average user

**Comment by noot** [comment]
looks good, I agree it would be nice to be able to configure it, but it doesn't seem like libp2p's `ConnManager` can be configured once it's initialized https://godoc.org/github.com/libp2p/go-libp2p-connmgr#NewConnManager
maybe putting it in the `configureMsg` for now would be good, so you could configure it on node startup?

**Comment by bkase** [comment]
This includes @noot 's changes so I can test in a qa-net

File: src/lib/cli_lib/default.ml [hunk 1]
```diff
@@ -1,4 +1,8 @@
 (* Default values for cli flags *)
 let work_reassignment_wait = 420000
 
+let max_connections = 50
+
+let validation_queue_size = 150
```
Comment (imeckler) [inline src/lib/cli_lib/default.ml:6]: What were these values before?

**Review by imeckler — APPROVED** [review]
LGTM

**Comment by bkase** [comment]
!ci-build-me

**Comment by bkase** [comment]
!ci-build-me

**Pull request merged**